### DISCUSSION TITLE
Hide Live Timing in events that are not live (with Replay Live Timing)

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#260] [Feature requests]
![image](https://user-images.githubusercontent.com/22590445/197410302-89115a02-b372-4f99-be0c-5d4fd4175cc1.png)

For a past event replay live timing is used, but when there is a live event at the same time, it might be better to hide Live Timing for the incorrect event.

**Comment by ManCaveMedia** [comment]
Would be even better imo if the live timing/live timing replay is only shown at the event it's for. Or is that effectively what you're suggesting?
